lib/polyfill/media_capabilities.js

```diff
@@ -201,6 +217,34 @@ shaka.polyfill.MediaCapabilities = class {
 
     return res;
   }
+
+  /**
+   * @param {!string} videoCodec
+   * @param {!string} audioCodec
+   * @param {!string} keySystem
+   * @return {!string}
+   * @static
+   */
+  static generateKeySystemCacheKey(videoCodec, audioCodec, keySystem) {
+    return `${videoCodec}#${audioCodec}#${keySystem}`;
+  }
+
+  /**
+   * @param {!string} videoCodec
+   * @param {!string} audioCodec
+   * @param {!string} keySystem
+   * @param {!MediaKeySystemAccess} result
+   * @static
+   */
+  static appendKeySystemCache(videoCodec, audioCodec, keySystem, result) {
```

Comment:
This is only used in a test that exercises it, but not elsewhere in the polyfill.

It appears the test in `describe('appendKeySystemCache'` is really a test of caching in `decodingInfo`.  Please move that test case to `describe('decodingInfo'` and migrate this function to a helper in the test itself.  You can use `@suppress {accessControls}` in the test helper annotations to access private members for the purposes of a test.

Reply:
The intention with `appendKeySystemCache` was to provide an api with which to pre-populate the cache externally.

Thanks again @joeyparrish for reviewing my PR :heart:.

Reply:
I have removed it.